blockapps-strato/azuredeploy.json

```diff
@@ -0,0 +1,231 @@
+            "dependsOn": [
+                "[variables('publicIPAddressPath')]",
+                "[variables('virtualNetworkPath')]",
+                "[variables('networkSecurityGroupPath')]"
+            ],
+            "properties": {
+                "networkSecurityGroup" : {
+                    "id" : "[variables('networkSecurityGroupId')]"
+                },
+                "ipConfigurations": [
+                    {
+                        "name": "[variables('ipConfigurationName')]",
+                        "properties": {
+                            "privateIPAllocationMethod": "Dynamic",
+                            "publicIPAddress": {
+                                "id": "[variables('publicIPAddressId')]"
+                            },
+                            "subnet": {
+                                "id": "[variables('subnetId')]"
+                            }
+                        }
+                    }
+                ]
+            }
+        },   
+        {
+            "location": "[variables('location')]",
+            "type": "Microsoft.Compute\/virtualMachines",
+            "name": "[variables('name')]",
+            "dependsOn": [
+                "[variables('storageAccountPath')]",
+                "[variables('networkInterfacePath')]"
+            ],
+            "properties": {
+                "provisioningState": 0,
+                "diagnosticsProfile": {
+                    "bootDiagnostics": {
+                        "enabled": false
+                    }
+                },
+                "networkProfile": {
+                    "networkInterfaces": [
+                        {
+                            "id": "[variables('networkInterfaceId')]"
+                        }
+                    ]
+                },
+                "osProfile": {
+                    "computerName": "[variables('name')]",
+                    "adminUsername": "[variables('username')]",
+                    "adminPassword": "[variables('password')]",
+                    "linuxConfiguration": {
+                        "disablePasswordAuthentication": "false"
+                    }
+                },
+                "storageProfile": {
+                    "osDisk": {
+                        "caching": "ReadWrite",
+                        "vhd": {
+                            "uri": "[variables('vhdURI')]"
+                        },
+                        "image": {
+                            "uri": "[variables('imageURI')]"
+                        },
+                        "createOption": "FromImage",
+                        "name" : "[concat(variables('name'), 'osDisk')]",
+                        "osType": "Linux"
+                    }
+                },
+                "hardwareProfile": {
+                    "vmSize": "[variables('vmSize')]"
+                }
+            },
+            "apiVersion": "2015-06-15"
+        }
+    ],
+    "parameters": {
```

Comment:
Purely cosmetic suggestion: For conformity with the rest of templates and the process folks are used to reading the templates, I suggest putting the resources section after parameters, variables